src/gensbi/diffusion/path/edm_path.py

```diff
@@ -1,3 +1,9 @@
+"""
+EDM (Elucidating Diffusion Models) probability path implementation.
```

Comment:
specify it comes from the paper Elucidating the Design Space of Diffusion-Based Generative Models

https://arxiv.org/abs/2206.00364